scalafmt-tests/src/test/resources/rewrite/AvoidInfix.stat

```diff
@@ -119,12 +122,14 @@ seq should have length 5
 >>>
 (seq should have).length(5)
 <<< excluded combinators in lhs 3
-1 to total foreach { _ =>
+1 to total foreach /* c1 */ { _ =>
   perform(_ + 1)
 }
 >>>
-(1 to total).foreach { _ =>
-  perform(_ + 1)
+(1 to total).foreach {
+  /* c1 */
```

Comment:
This comment moving through bracket looks strange but not critical

Reply:
that was the request in one of the 4 original bug reports